source/SIL.AppBuilder.Portal.Frontend/src/lib/auth0.ts

```diff
@@ -80,6 +86,15 @@ let lockInstance: Auth0LockStatic;
 export function getAuth0LockInstance(options = {}): Auth0LockStatic {
   if (lockInstance === undefined) {
     lockInstance = new Auth0Lock(auth0Env.clientId, auth0Env.domain, {
+      // TODO: pull language form i18next
+      language: 'en',
```

Comment:
locale is available from google-auth2 provider jwt.

Reply:
this is before we have the jwt.